clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -6105,6 +6105,33 @@ bool TokenAnnotator::canBreakBefore(const AnnotatedLine &Line,
       return false;
   }
 
+  // We only break before r_brace if there was a corresponding break before
+  // the l_brace, which is tracked by BreakBeforeClosingBrace.
```

Comment:
```suggestion
  // We only break before an r_brace if there was a corresponding break after
  // the matching l_brace, which is tracked by BreakBeforeClosingBrace.
```

Reply:
Actually, this comment is outdated. I've pushed your suggestion with a little bit more correction.